genai/controlled_generation/ctrlgen_with_class_schema.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def generate_content() -> str:
+    # [START googlegenaisdk_ctrlgen_with_class_schema]
+    from google import genai
+    from pydantic import BaseModel, TypeAdapter
+
+    class Recipe(BaseModel):
+        recipe_name: str
+        ingredients: list[str]
+
+    client = genai.Client()
+    response = client.models.generate_content(
+        model="gemini-2.0-flash-001",
+        contents="List a few popular cookie recipes.",
+        config={
+            "response_mime_type": "application/json",
+            "response_schema": list[Recipe],
+        },
```

Comment:
Can you use the `GenerateContentConfig` class?

Reply:
We will have another example to focus on `GenerateContentConfig`. So being straight forward here (less jargons).